app/src/main/java/org/mozilla/fenix/FenixApplication.kt

```diff
@@ -100,6 +99,28 @@ open class FenixApplication : Application() {
             ExperimentsMetrics.activeExperiment.set(branchName)
         }
 
+        // When the `fenix-etp-2019-10-01` experiment is active, set up ETP settings and GV policy.
+        // Note that this will not take effect the first time the application has launched,
+        // since there won't be enough time for the experiments library to get a list of experiments.
+        // It will take effect the second time the application is launched.
+        Experiments.withExperiment("fenix-etp-2019-10-01") { branchName ->
```

Comment:
Can you add this to the `Experiments.kt` file? There's a pattern of how experiments are created and used there, so we can keep them all standard.

Reply:
I talked with @mythmon about this and IIUC that file uses Fretboard but we want to align closer with the way desktop does it through Experimenter for this experiment.